app.rsa

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIICXQIBAAKBgQDLnRPJJJPloWbjLbaSx25C+l20i8e6fahACjc3qsZOTacelDRS
+ZTXO4qxKOZGuXIBI56rmFdfIboIfv2uknpsdl7E/g/D45KHKwLiiYxf1qMtNfOVj
+H8m3nPKCVjDfAJ4XR+8YXpLu9c2RnrAETvHTK35Q+1xUHCI5Vyp9o1swrwIDAQAB
+AoGBALoecTGRz+c9bBEJfjQZPD7ZIMKiawWbSyqOq/6kifSVosee7VzLyy27oYWP
+rgzV3SXHLj53hPFfom5myXyGxn3a5fqJSQVeFvTAt472BnrzCCw8ogeOpQ6H/6Xs
+pHijrK4gh/M2GfDAWFJjAKuLicELQ3A2cCIyu0SraaJer+PBAkEA6YKbGC0WwJdi
+k79EYLCihrB/xexBcp/8gY//zAhLyqD0mPe5NC/4ml9tBSA0EoQ3IOqoFyLddBGc
+J+Edu5R5vwJBAN85WF3nfqklXBVd4MjaNKP+NZjCZcPpaWmymgH7W36CuuvLAb26
+JwD+JvHNWyUh/oMm40iCnuhqW26OhL+5pRECQAIaKOdI0KGfO0X1PNpZ/LyVrjDL
+9bLgedd4wmbF1FjYphsSb9pmbCiHn2WiapWYFx72vdinebLnza4WOrxLk+UCQHQv
+RdZCioOBhYc9gKUqJeIBi2YCMLOixpIUOuKzMLENXDskQ7i7rldu8VhD2Z/fLLiY
+wg3YptszmHlXxrJ/7vECQQCzQ5qylgVoaTzuVc1QVaxGhUN6wmNQPjRA5Hba1rfh
+7QmEHtvkr7BwCitm6J+86WXhp9gy7/CioTbMOvfilUyy
+-----END RSA PRIVATE KEY-----
```

Comment:
@marcotroisi Yes absolutely - I added this for testing , we should have a key pair basically for each service and store them by service name in some secure folder